pkg/maps/ctmap/ctmap.go

```diff
@@ -412,13 +412,14 @@ func maps(e CtEndpoint, ipv4, ipv6 bool) []*Map {
 			result = append(result, NewMap(MapNameAny6Global, MapTypeIPv6AnyGlobal))
 		}
 	} else {
+		idStr := fmt.Sprintf("%05d", int(e.GetID()))
```

Comment:
nit: nit: `bpf.LocalMapName()` will add the names together with the proper "%05d", without duplicating the number of places where this `%05d` logic lives in the codebase.

Reply:
Updated!